Xamarin.Forms.Core/IBorderOrCornerRadius.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Xamarin.Forms
+{
+	/// <summary>
+	/// this exists purely to mark classes using the legacy BorderRadius Property
+	/// </summary>
+	interface IBorderOrCornerRadius
+	{
+		/// <summary>
+		/// Flag to prevent overwriting the value of CornerRadius
+		/// </summary>
+		bool cornerOrBorderRadiusSetting { get; set;}
```

Comment:
I added this because I didn't want this property contaminating new uses of IBorderElement.

Reply:
I am not disappointed